OnlineStore.UI/Controllers/CategoryController.cs

```diff
@@ -0,0 +1,69 @@
+﻿using OnlineStore.BLL.Services.Interfaces;
+using OnlineStore.BLL.DTO;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
+
+namespace OnlineStore.UI.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CategoryController : ControllerBase
```

Comment:
check in swagger your contorller names should start with a small letter accroding to REST uri naming conventions